pkg/appgw/health_probes.go

```diff
@@ -162,6 +163,16 @@ func (c *appGwConfigBuilder) generateHealthProbe(backendID backendIdentifier) *n
 		probe.Path = to.StringPtr(strings.TrimRight(*probe.Path, "*"))
 	}
 
+	if customPath, err := annotations.BackendProbePath(backendID.Ingress); err == nil {
+		if matched, reErr := regexp.MatchString(`^\/([.a-zA-Z0-9-_~\?\#\[\]\@\:\$\&\(\)\*\+\,\;\=]+/??)*$`, customPath); matched == true && reErr == nil {
+			probe.Path = to.StringPtr(customPath)
+			glog.V(5).Infof("Created custom path %s for ingress %s/%s probe", *probe.Path, backendID.Ingress.Namespace, backendID.Ingress.Name)
+		} else {
+			probe.Path = to.StringPtr("/")
+			glog.V(5).Infof("Custom path %s for ingress %s/%s probe is not valid! Using '/' instead.", customPath, backendID.Ingress.Namespace, backendID.Ingress.Name)
```

Comment:
use warning instead of info

Reply:
Resolved in [
d2ac719 ](https://github.com/Azure/application-gateway-kubernetes-ingress/pull/1006/commits/d2ac719c07d1ec7c3ab1701ee70a07784a86e9de)